src/services/codefixes/importFixes.ts

```diff
@@ -0,0 +1,393 @@
+﻿/* @internal */
+namespace ts.codefix {
+
+    registerCodeFix({
+        errorCodes: [Diagnostics.Cannot_find_name_0.code],
+        getCodeActions: (context: CodeFixContext) => {
+            const sourceFile = context.sourceFile;
+            const checker = context.program.getTypeChecker();
+            const allSourceFiles = context.program.getSourceFiles();
+            const useCaseSensitiveFileNames = context.host.useCaseSensitiveFileNames ? context.host.useCaseSensitiveFileNames() : false;
+
+            const token = getTokenAtPosition(sourceFile, context.span.start);
+            const name = token.getText();
+            const allActions: CodeAction[] = [];
+
+            const allPotentialModules = checker.getAmbientModules();
+            for (const otherSourceFile of allSourceFiles) {
+                if (otherSourceFile !== sourceFile && otherSourceFile.symbol) {

```

Comment:
Is the presence of ".symbol" an indicator this is a module then? Is the helpful function `isExternalModule` not usable here? (It is more readable intuitive if so).